_includes/tutorials/hopping-windows/flinksql/code/src/test/java/io/confluent/developer/FlinkSqlHoppingWindowTest.java

```diff
@@ -18,6 +18,7 @@ public void testHoppingWindows() throws Exception {
     streamTableEnv.executeSql(getResourceFileContents("populate-temperature-readings.sql"));
     streamTableEnv.executeSql(getResourceFileContents("create-temperature-by-10min-window.sql.template",
```

Comment:
now that line 19 is just the CREATE TABLE statement, tack on `.await()` that should prevent race